### ISSUE TITLE
Issues opening Item Sheets and Item Summary in current master branch

### ISSUE THREAD

**Issue by glont** [foundryvtt/pf2e#6478]
While working on #6476 I found some issues with opening Items, fixed them locally, but I'm not sure if I did anything bad. The problems where there without any active modules:

# Items can not be opened

In _[/src/module/item/sheet/base.ts](https://github.com/foundryvtt/pf2e/blob/d30ec6c1e41a98d1813df7d96851d0f32424c19d/src/module/item/sheet/base.ts#L95)_, when trim() is called on gm notes, it crashes, because that value is null.

`enrichedContent.gmNotes = await TextEditor.enrichHTML(itemData.system.description.gm.trim(), {`
`enrichedContent.gmNotes = await TextEditor.enrichHTML(itemData.system.description.gm?.trim(), {`

Not sure this is a good solution though.

# Items summary can not be opened in sheets

As far as I can tell its missing `import { gsap } from "gsap";` in _[/src/module/actor/sheet/item-summary-renderer.ts](https://github.com/foundryvtt/pf2e/blob/d30ec6c1e41a98d1813df7d96851d0f32424c19d/src/module/actor/sheet/item-summary-renderer.ts)_, which seems to fixe this. 

Again this may not be a good solution.

**Comment by stwlam** [comment]
Would you mind providing reproduction steps?

**Comment by glont** [comment]
I currently use the foundry desktop app for local development, maybe that creates issues, but I just tried to create a new world and the issues do not happen then. Then after that it also does not happen in the old world.

I'm not exactly sure why, I actually checked in the old world just before creating this issue, it basically happened when trying to open any Equipment from the Compendium Browser.

I guess this can be closed this, I think something did not correctly work when I first used `npm run build:dev`...
